src/less/textbox/base/textbox.less

```diff
@@ -6,24 +6,24 @@
     color: @textbox-control-color;
     font-size: @textbox-control-font-size;
     position: relative;
-}
-
-span.textbox {
-    display: inline-block;
 
     button.icon-btn {
         padding: 0;
         position: absolute;
         right: 0;
 
-        .textbox__icon {
+        svg.icon {
             left: 0;
             right: 0;
-            width: @textbox-icon-width;
+            width: @textbox-actionable-icon-width;
```

Comment:
@raveenakothapally So I think what @agliga is saying is to try dropping this width property, and instead just falling back to the width that is defined by the icon modifier, e.g. `svg.icon--messages`. `svg.icon--profile`, etc.

Reply:
It shrinks if we do not assign a specific width.

The final style that it has is derived from `.textbox svg` which overrides `svg.icon--messages` that has 1em and 24px respectively as widths. 1em within a button is too small whereas 24px (If we manage to increase the specificity) will be too large.

Reply:
Ah, I see. I didn't realize it was coming from `.textbox svg`. I think perhaps we can just leave it as it is now. It's good that we got rid of `textbox__icon`.